### DISCUSSION TITLE
Extensions folder

### DISCUSSION THREAD

**Discussion by MilanNedoma** [bonsai-rx/bonsai#1579] [General]
Hi!

Apologies for the basic question, however I am trying to get DLC working with Bonsai and I have met a step of the setup that I cannot complete. When it says: "After downloading the native TensorFlow binary and cuDNN, you can follow these steps to get the required native files into the Extensions folder of your local Bonsai install". I have tried following the readme folder, but I simply cannot find the extensions folder for Bonsai anywhere. There is only a .vsix file named "Install Bonsai VS Extensions" and I cannot even open that one.

Could someone please tell me what exactly is meant by this "Extensions" folder?

Thank you very much in advance!

UPDATE:
Managed to open the file mentioned above, but I still don´t know where to put the TensorFlow and cuDNN .dll files.

**Comment by ncguilbeault** [comment]
Hi there, the Extensions folder should be in the location of the install directory of Bonsai, so wherever the Bonsai.exe file is located. If you installed the portable version of Bonsai (the Bonsai.zip version), the Extensions folder will be in the extracted folder. If you installed Bonsai globally by running the Bonsai installer, then it will be located in the App folder. This is typically located in C:\Users\$USERNAME\AppData\Local\Bonsai where you replace $USERNAME with your actual username.

**Reply by MilanNedoma** [reply]
Thank you so much! I have managed to locate the folder now.
